build/storage/core/host-target/pci_devices.py

```diff
@@ -144,8 +144,22 @@ def _match_namespaces_to_volumes(
 
 
 def _find_namespaces_in_dev(namespaces_in_sysfs: list[str]) -> set[Volume]:
+    from string import digits
+
     namespace_dev_paths = set()
     for namespace_in_sysfs in namespaces_in_sysfs:
-        namespace_dev_path = os.path.join("/dev", os.path.basename(namespace_in_sysfs))
+        dev = os.path.basename(namespace_in_sysfs)
+        namespace_dev_path = os.path.join("/dev", dev)
+        if not os.path.exists(namespace_dev_path) and "c" in namespace_dev_path:
+            c_position = dev.find("c")
```

Comment:
Is this universally working method? I wonder if we can find this info at a higher abstraction level? But if it works reliably I am fine.

Reply:
Added TODO